test/UnitTest/Microsoft.AspNet.OData.Test.Shared/Formatter/Serialization/ODataResourceSerializerTests.cs

```diff
@@ -275,7 +275,7 @@ public void WriteObjectInline_Calls_CreateNavigationLink_ForEachExpandedNavigati
             // Arrange
             SelectExpandNode selectExpandNode = new SelectExpandNode
             {
-                ExpandedNavigationProperties =
```

Comment:
>ExpandedNavigationProperties [](start = 16, length = 28)

Shall we add ExpandedNavigationPropertiesWithExpandedItem and leave ExpandedNavigationProperties unchanged?

Reply:
The unit test will fail for ExpandedNavigationProperties. 

---
In reply to: [282618354](https://github.com/OData/WebApi/pull/1724#discussion_r282618354) [](ancestors = 282618354)